app/views/users/mailer/password_change.html.erb

```diff
@@ -0,0 +1,3 @@
+<p>Hello <%= @resource.email %>!</p>
+
+<p>We're contacting you to notify you that your password has been changed.</p>
```

Comment:
I know this one wasn't part of the ticket, but maybe we can change this one, too, to use `first_name`, and in the second bit maybe have something like, "If you didn't change your password, please use the Reset Password functionality to recover your password ASAP." Users will get this one as part of the password reset flow.